### ISSUE TITLE
Build fails on headless (x11-less) linux machines

### ISSUE THREAD

**Issue by roblabla** [NationalSecurityAgency/ghidra#4761]
**Describe the bug**
On headless machines (such as the ones used by github CI), build ghidra fails on the `:Docking:buildHelp FAILED` step, with the following error:

```
Error: Exception in thread "main" java.awt.AWTError: Can't connect to X11 window server using ':0.0' as the value of the DISPLAY variable.
	at java.desktop/sun.awt.X11GraphicsEnvironment.initDisplay(Native Method)
	at java.desktop/sun.awt.X11GraphicsEnvironment$1.run(X11GraphicsEnvironment.java:101)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
	at java.desktop/sun.awt.X11GraphicsEnvironment.<clinit>(X11GraphicsEnvironment.java:60)
	at java.desktop/sun.awt.PlatformGraphicsInfo.createGE(PlatformGraphicsInfo.java:36)
	at java.desktop/java.awt.GraphicsEnvironment$LocalGE.createGE(GraphicsEnvironment.java:93)
	at java.desktop/java.awt.GraphicsEnvironment$LocalGE.<clinit>(GraphicsEnvironment.java:84)
	at java.desktop/java.awt.GraphicsEnvironment.getLocalGraphicsEnvironment(GraphicsEnvironment.java:106)
	at java.desktop/sun.awt.X11.XToolkit.<clinit>(XToolkit.java:224)
	at java.desktop/sun.awt.PlatformGraphicsInfo.createToolkit(PlatformGraphicsInfo.java:40)
	at java.desktop/java.awt.Toolkit.getDefaultToolkit(Toolkit.java:599)
	at java.desktop/sun.swing.SwingUtilities2.getSystemMnemonicKeyMask(SwingUtilities2.java:2206)
	at java.desktop/javax.swing.plaf.basic.BasicLookAndFeel.initComponentDefaults(BasicLookAndFeel.java:1100)
	at java.desktop/javax.swing.plaf.metal.MetalLookAndFeel.initComponentDefaults(MetalLookAndFeel.java:446)
	at java.desktop/javax.swing.plaf.basic.BasicLookAndFeel.getDefaults(BasicLookAndFeel.java:153)
	at java.desktop/javax.swing.plaf.metal.MetalLookAndFeel.getDefaults(MetalLookAndFeel.java:1566)
	at java.desktop/javax.swing.UIManager.setLookAndFeel(UIManager.java:593)
	at java.desktop/javax.swing.UIManager.setLookAndFeel(UIManager.java:635)
	at java.desktop/javax.swing.UIManager.initializeDefaultLAF(UIManager.java:1412)
	at java.desktop/javax.swing.UIManager.initialize(UIManager.java:1525)
	at java.desktop/javax.swing.UIManager.maybeInitialize(UIManager.java:1491)
	at java.desktop/javax.swing.UIManager.getInstalledLookAndFeels(UIManager.java:428)
	at java.desktop/javax.swing.UIManager.installLookAndFeel(UIManager.java:471)
	at java.desktop/javax.swing.UIManager.installLookAndFeel(UIManager.java:490)
	at generic.theme.ApplicationThemeManager.installFlatLookAndFeels(ApplicationThemeManager.java:375)
	at generic.theme.ApplicationThemeManager.doInitialize(ApplicationThemeManager.java:76)
	at generic.theme.ApplicationThemeManager.initialize(ApplicationThemeManager.java:66)
	at help.GHelpBuilder$1.initializeApplication(GHelpBuilder.java:73)
	at ghidra.framework.Application.initializeApplication(Application.java:115)
	at help.GHelpBuilder.main(GHelpBuilder.java:81)

FAILURE: Build failed with an exception.
```

The problem started to appear sometime after e13f22dea (last known working commit). I suspect commit 45c52e3cb97e44305f34d6007efd56d5368cd257 is responsible for the problem.

This makes it very annoying to package for distributions.

**To Reproduce**

On a headless machine, run `gradle build -x ip buildGhidra`.

**Expected behavior**
The build to work.

**Screenshots**

**Environment (please complete the following information):**
 - OS: Linux
 - Java Version: 17
 - Ghidra Version: Latest master
 - Ghidra Origin: locally built

**Comment by ryanmkurtz** [comment]
Thanks, we have this fixed internally.  Just waiting on the tests to pass before pushing it up to GitHub.

**Comment by ryanmkurtz** [comment]
Fixed by 75ddd08bbd0fe87451f49896388f7cdb1218039f
